keps/4400-workload-priority-preemption-policy/kep.yaml

```diff
@@ -0,0 +1,26 @@
+title: WorkloadPriorityClass Preemption Policy
+kep-number: 4400
+authors:
+  - "@jenny-yang-ai"
+status: provisional
+creation-date: 2025-07-22
+reviewers:
+  - "@kannon92"
+  - "@tenzen-y"
+approvers:
+  - "@kannon92"
+  - "@tenzen-y"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
```

Comment:
@tenzen-y @mimowo 

This seems to not be too involved. Do we still want alpha for it?

I'm not sure if we follow the same pattern as k8s that new APIs require an alpha.